Core/clim-core/bordered-output.lisp

```diff
@@ -456,8 +456,29 @@
                           (draw-line* stream left bottom right bottom
```

Comment:
In `examples`, there is a bordered output test and a dedicated underlining test (within drawing tests?). One or both should test this feature.

Reply:
done